api/routes/books.js

```diff
@@ -1,10 +1,18 @@
 const router = require('express').Router()
 const Book = require('../models/book')
+const User = require('../models/user')
 
 router.get('/', async (req, res, next) => {
-  const status = 200
-  const response = await Book.find().select('-__v')
-  
+
+  validToken = tokenValidate(req);
+  const user = await User.findOne({ _id: validToken.id })
+  if (user.admin == true) { 
+
+    const status = 200
+    const response = await Book.find().select('-__v')
+  } else {
+    status = 401
+  }
```

Comment:
I'd be surprised if this worked? For this route, it's OK to let anyone request this information. But, it also looks like `response` won't be defined in the case the user is not an admin.